statusHandler/presenter/chainInfoGetters.go

```diff
@@ -51,3 +55,89 @@ func (psh *PresenterStatusHandler) GetNumConnectedPeers() uint64 {
 func (psh *PresenterStatusHandler) GetCurrentRound() uint64 {
 	return psh.getFromCacheAsUint64(core.MetricCurrentRound)
 }
+
+// CalculateSynchronizationEstimation will calculate and return an estimation of
+// the time required for synchronization in a human friendly format
+func (psh *PresenterStatusHandler) CalculateSynchronizationEstimation() string {
+	currentBlock := psh.GetNonce()
+
+	psh.mutEstimationTime.Lock()
+	diffTime := time.Now().Sub(psh.startTime).Seconds()
+	blocksSynchronized := currentBlock - psh.startBlock
+	psh.mutEstimationTime.Unlock()
+
+	if blocksSynchronized == 0 {
```

Comment:
I think an estimation could be calculated even if no blocks were synchronized

Reply:
The estimate is calculated in another way.